ScoutSuite/providers/aws/resources/ec2/snapshots.py

```diff
@@ -18,6 +18,9 @@ def _parse_snapshot(self, raw_snapshot):
         raw_snapshot['id'] = raw_snapshot.pop('SnapshotId')
         raw_snapshot['name'] = get_name(raw_snapshot, raw_snapshot, 'id')
         raw_snapshot['public'] = self._is_public(raw_snapshot)
+        raw_snapshot['arn'] = 'arn:aws:ec2:{}:{}:snapshot/{}'.format(self.get('region'),
+                                                                             self.facade.owner_id,
```

Comment:
I think that, as snapshots could be public, in this case it would be better to get the owner ID from the raw_snapshot object

Reply:
Agreed - I'd think this is included in the `raw_snapshot`?